package.json

```diff
@@ -68,11 +69,17 @@
     "superagent": "^3.2.1",
     "through2": "^2.0.3"
   },
+  "optionalDependencies": {
+    "pelias-interpolation": "*"
```

Comment:
yeah i wasn't sure how to handle this since it doesn't actually exist in npm. we would need to either remove it or publish the module to npm so this works.